### PR TITLE
feat(theme-docs): display updatedAt date in bottom of page

### LINKED ISSUE #535: Implement createdAt and UpdateAt in Docs Theme
### Describe the solution you'd like
It would be helpfull if you could see the values of createdAt and updatedAt at every page in the docs. Maybe it would be even more helpfull to see the updatedAt value since a User could see at first glance if something ist new. 
Since these informations are available at every page we could show them. 

AFAIK there is no build in function at the moment. 

I have wirtten a small component which would display it in the prev next page area. It was experimental. Before I continue I would like to hear your opinion on this. 

### Describe alternatives you've considered
Maybe beforeRender

### PR DESCRIPTION
**Pull request by Geminii**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
Display updateAt date into the bottom of the page (on the right)
Define translation and a default fallback language to EN. 
Resolves #535

### CODE REVIEW

File: packages/theme-docs/src/i18n/en-US.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ export default {
     title: 'On this page'
   },
   article: {
-    github: 'Edit this page on GitHub'
+    github: 'Edit this page on GitHub',
+    updatedAt: 'Updated at'
```
Comment (NozomuIkuta) [inline packages/theme-docs/src/i18n/en-US.js:10]: After this PR is merged, I will submit a PR to add Japanese translation for `updatedAt` text. 😄 

Or, if authors and collaborators want it to be included in one commit, please set:

```javascript
{
  updatedAt: '更新日'
}
```

Comment (Geminii) [reply packages/theme-docs/src/i18n/en-US.js:10]: @NozomuIkuta I integrated your translation, thanks for it 👍

**Comment by Geminii** [comment]
@Atinux any feedback about this ?

**Comment by benjamincanac** [comment]
Thanks @Geminii! It's really nice 👍 I just removed the unused `AppGithubLink` component.

**Review by benjamincanac — APPROVED** [review]

**Pull request merged**